test/bpk-button-v2-fail.jsx

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars,import/no-unresolved,import/extensions */
+// backpack/use-button-v2
+import BpkButton from '@skyscanner/backpack-web/bpk-component-button';
```

Comment:
After add the rule, it would show warning for this test component as follow

` 11:8  warning  Use BpkButtonV2 instead of BpkButton. BpkButton is deprecated and will be removed soon  backpack/use-button-v2`

<img width="928" height="201" alt="image" src="https://github.com/user-attachments/assets/b55d1ae0-ccf8-48d6-97c0-d499396006fa" />